cache/cache.go

```diff
@@ -32,9 +33,15 @@ var ctx = context.Background()
 
 func CreateNewRedisClient(config RedisConfig) *Client {
 	c := Client{}
-	// Try to connect to redis, check if operation mode is cluster, if yes then try all possible ways to connect.
-	if config.OperationMode == ModeCluster {
-		if config.Url != "" {
+
+	if !config.Enabled {
+		fmt.Println("P#1UF5RK - Enable redis to acquire a connection object.")
+		os.Exit(exitCode.RedisConnectionFailed)
```

Comment:
Return nil instead please. Else a simple configuration will cause the entire program to end!